cli/src/auth.rs

```diff
@@ -183,15 +183,18 @@ where
 	T: Serialize + ?Sized,
 {
 	let dec = serde_json::to_string(value).expect("expected to serialize");
-	encrypt(&dec)
+	if std::env::var("VSCODE_CLI_DISABLE_ENCRYPT").is_ok() {
+		return dec;
+	}
+	encrypt(&dec);
```

Comment:
```suggestion
	encrypt(&dec)
```

fixes the build error